mtp_noms_ops/templates/settings/job-information.html

```diff
@@ -0,0 +1,97 @@
+                            {% endfor %}
+                        {% endwith %}
+                    </div>
+                    </fieldsetclass="mtp-conditional-container">
+                    {% endwith %}
+                    {% with field=form.prison_estate choices=form.prison_estate.field.choices %}
+                    <fieldset>
+                        <legend id="{{ field.id_for_label }}-label">
+                            <strong class="heading-large">{{ field.label }}</strong>
+                        </legend>
+
+                        <div class="form-group {% if field.errors %}form-group-error{% endif %}">
+                            <p class="form-hint">{{ field.help_text }}</p>
+                            {% include 'mtp_common/forms/field-errors.html' with field=field only %}
+                            {% with initial=value|default:field.value %}
+                                {% for value, label in choices %}
+                                    <div class="multiple-choice">
+                                        <input id="{{ field.html_name }}-{{ value }}" type="radio" name="{{ field.html_name }}" value="{{ value }}" {% if value == initial %}checked{% endif %}>
+                                        <label for="{{ field.html_name }}-{{ value }}">{{ label }}</label>
+                                    </div>
+                                {% endfor %}
+                            {% endwith %}
+                            </div>
+                        </fieldset>
+                        {% endwith %}
+                        {% with field=form.tasks %}
+                        <fieldset>
+                            <legend id="{{ field.id_for_label }}-label">
+                                <strong class="heading-large">{{ field.label }}</strong>
+                            </legend>
+                            <div id="{{ field.id_for_label }}-wrapper" class="form-group {% if field.errors %}form-group-error{% endif %}">
+                                <p class="form-hint">{{ field.help_text }}</p>
+                                {% include 'mtp_common/forms/field-errors.html' with field=field only %}
+
+                                <textarea id="{{ field.id_for_label }}" class="form-control {% if field.errors %}form-control-error{% endif %} {{ input_classes }}" name="{{ field.html_name }}" rows="4" cols="50" {% for attr, attr_val in attrs.items %} {{ attr }}="{{ attr_val }}" {% endfor %}>{{ value|default:field.value|default_if_none:'' }}</textarea>
+                            </div>
+                        </fieldset>
+                        {% endwith %}
+                        <div class="form-group">
+                            <button type="submit" class="button">
+                                {% trans 'Continue' %}
+                            </button>
+                        </div>
+                    </form>
+
+                </div>
+            </div>
+
+        {% endblock %}
```

Comment:
is this the end of `{% block inner_content %}`? it's much more deeply indented here, so i wonder if there are missing closed elements or something iffy going on with the indentation further up